assets/docs/adityakane2001/adityakane2001.md

```diff
@@ -0,0 +1,15 @@
+# Publisher AdityaKane2001
```

Comment:
If you want "AdityaKane2001" to have capital letters, you need to change the directory name across all these markdown files to match (and in this case, the .md file name too)

Reply:
Actually, sorry for this, by convention we prefer lowercase publisher names. Could you please swap everything back to lowercase?

Reply:
@MorganR 

I want to keep `adityakane2001` everywhere, following the repo convention. In that case, I should keep all filenames and contents inside them as `adityakane2001` , right?

Reply:
Correct, this looks correct now.